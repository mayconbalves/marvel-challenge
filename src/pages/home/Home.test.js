import { render, screen } from '@testing-library/react';
import Home from './';

test('renders title of home', () => {
  render(<Home />);
  const linkElement = screen.getByText(/home/i);
  expect(linkElement).toBeInTheDocument();
});
