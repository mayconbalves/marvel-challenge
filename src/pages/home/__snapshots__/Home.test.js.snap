// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correct title element 1`] = `
<div
  class="sc-bdfBwQ kSwYoH"
  color="#fff"
>
  <div
    class="sc-dlfnbm kuxBMX"
  >
    <div
      class="sc-hKgILt grZqev"
    >
      <div
        class="sc-eCssSg jgsGZg"
      >
        <img
          alt="logo"
          src="logo.png"
        />
      </div>
      <div
        class="sc-eCssSg jgsGZg"
      >
        <h1
          class="sc-jSgupP cnfwtI"
        >
          Explore o universo
        </h1>
        <p
          class="sc-gKsewC hBazWe"
        >
          Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aquelas que você descobrirá em breve!
        </p>
      </div>
      <div
        class="sc-eCssSg jgsGZg"
      >
        <input
          class="sc-gsTCUz cntDrA"
          name="hero"
          placeholder="Procure por heróis"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="sc-eCssSg jgsGZg"
    >
      <div
        class="sc-iBPRYJ inxKGS"
      >
        <div>
          <p>
            Encontramos 0 heróis
          </p>
        </div>
        <div
          class="sc-fubCfw bUVNAC"
        >
          <img
            alt="super herói"
            src="superhero.png"
          />
          <p>
            Ordernar por nome A/Z
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-pFZIQ joIIuf"
    />
  </div>
</div>
`;
